### PR TITLE
Update configure-operator-defaults.md

### PR DESCRIPTION
**Pull request by pstack2021**

### CODE REVIEW

File: site/kubernetes/operator/configure-operator-defaults.md [hunk 1]
```diff
@@ -15,16 +15,82 @@ See the License for the specific language governing permissions and
...
 
-## <a id='parameters' class='anchor' href='#parameters'>Configurable Parameters</a>
+When the Cluster Operator is deployed, update the manifest by completing the following steps: 
+
+1. Run: 
+<pre class="lang-bash">
+kubectl -n rabbitmq-system edit deployment rabbitmq-cluster-operator
+</pre>
+
+2.  Add the environment variables you wish to change. In the following example, the `OPERATOR_SCOPE_NAMESPACE` environment variable is added and setto `custom-namespace` for the cluster operator.
```
Comment (michaelklishin) [inline site/kubernetes/operator/configure-operator-defaults.md]: A typo: "setto" => "set to"

Comment (pstack2021) [reply site/kubernetes/operator/configure-operator-defaults.md]: Thanks @michaelklishin will fix

**Review by michaelklishin — APPROVED** [review]
LGTM except for one typo

**Comment by pstack2021** [comment]
Thanks for reviewing so quickly @michaelklishin if you can answer my queries when you have time that would be great.

**Pull request merged**